sdk/storage_blobs/src/blob/operations/put_block_list.rs

```diff
@@ -79,6 +80,7 @@ impl PutBlockListBuilder {
 pub struct PutBlockListResponse {
     pub etag: String,
     pub last_modified: OffsetDateTime,
+    #[cfg(feature = "md5")]
```

Comment:
Is it idiomatic to change the shape of a model based on a feature? Should this maybe just be an `Option<ConsistencyMD5>` and always be `None` otherwise?

Reply:
Probably not, though my implementation was based on the service implementation.  If the md5 is not sent in the request, the service should not send it in a response.

Happy to adjust the PR though.

Reply:
Generally, we want all our SDKs to have the same public APIs regardless of targets/features/whatever. In .NET, for example, our public APIs are the same regardless of which TFMs we target.